DataStructures.Tests/ArrayBasedCircularQueueTests.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using DataStructures.ArrayBasedCircularQueue;
+using NUnit.Framework;
+
+namespace DataStructures.Tests
+{
+    public class ArrayBasedCircularQueueTests
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6301193)